lib/modules/apostrophe-assets/index.js

```diff
@@ -575,6 +575,17 @@ module.exports = {
         var last = chain[chain.length - 1];
         var from = self.getAssetRoot() + '/public/modules/' + name;
         var to = last.dirname + '/public';
+        var scope = to.match(/@.+?(?=\/)/);
+
+        // Create symlink folder for scoped package, if it does not exist.
+        if (scope) {
+          var symlink = self.getAssetRoot() + '/public/modules/' + scope[0];
```

Comment:
This isn't a symlink, it's a folder.

Repeating the path actually increases the risk of bugs, and the use of a variable will make code faster if anything, as it avoids repeated computation (although performance is not really at issue in this method - correctness and readability are). My code earlier in the method where I repeat that path is rather lazy.